sdk/keyvault/test-resources.json

```diff
@@ -6,19 +6,19 @@
       "type": "string",
       "defaultValue": "[resourceGroup().name]",
       "metadata": {
-          "description": "The base resource name."
+          "description": "The base resource name. This field gets automatically generated for us through our CI pipelines. It is required to replace this default value with any combination of alphanumeric characters that you might prefer."
```

Comment:
Things I'd call out about this field: 
* `baseName` is used to ensure unique names of resources
* Automatically generated by CI pipelines (DONE) 
* Replace with alphanumeric characters (DONE) (It's possible to use non-alphanumeric characters in some resource names, like dashes in KV names, but keeping it simple will reduce confusion) 
* A safe upper limit on the number of characters is [17](https://github.com/Azure/azure-sdk-for-js/blob/master/eng/New-TestResources.ps1#L156)... The `baseName` may only be a subset of the characters used in the naming (e.g. [storage account names may preface with their own character](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/test-resources.json#L41)) so the length constraints here are different for every template depending on the resource used.

Reply:
Pushed a fix to this! Let me know if it's ok now.